From 928d856d87f551e9f653792d6979383965974066 Mon Sep 17 00:00:00 2001
From: Louis Scalbert <louis.scalbert@6wind.com>
Date: Fri, 4 Sep 2026 14:39:00 +0200
Subject: [PATCH] bgpd: fix bgp flag conflict

BGP_FLAG_SOFT_VERSION_CAPABILITY_NEW and
BGP_FLAG_BESTPATH_USE_IMPORTED_ATTRS have the same value.

Signed-off-by: Louis Scalbert <louis.scalbert@6wind.com>
(cherry picked from commit 49908328e1a2954148809f38add544ee46e96b44)
---
 bgpd/bgpd.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/bgpd/bgpd.h b/bgpd/bgpd.h
index 1a497497fd35..9d86658307d4 100644
--- a/bgpd/bgpd.h
+++ b/bgpd/bgpd.h
@@ -759,11 +759,10 @@ struct bgp {
 #define BGP_FLAG_VRF_MAY_LISTEN		    (1ULL << 44)
 #define BGP_FLAG_SOFT_VERSION_CAPABILITY_NEW (1ULL << 45)
 #define BGP_FLAG_USE_RECURSIVE_WEIGHT (1ULL << 46)
-
 /* Use current (imported) path's attributes instead of source path's attributes
  * for bestpath comparison of imported paths.
  */
-#define BGP_FLAG_BESTPATH_USE_IMPORTED_ATTRS (1ULL << 45)
+#define BGP_FLAG_BESTPATH_USE_IMPORTED_ATTRS (1ULL << 47)
 
 	/* BGP default address-families.
 	 * New peers inherit enabled afi/safis from bgp instance.
